server/models/user.model.js

```diff
@@ -1,29 +1,35 @@
-/**
- * User Schema
- */
-module.exports = (sequelize, DataTypes) => {
-    const User = sequelize.define('User', {
-        id: {
-            type: DataTypes.INTEGER,
-            autoIncrement: true,
-            primaryKey: true,
+import { Schema } from 'mongoose';
```

Comment:
Do we want `createdAt` and `updatedAt` fields?

Reply:
Mongo ObjectIDs have the createdAt date embedded in them.
We should implement updatedAt to reach parity with pre-mongo notifications service

Reply:
done!